### ISSUE TITLE
Broken links to other 18MR posts in comment section (disqus-related)

### ISSUE THREAD

**Issue by seemala** [18mr/18mr#7]
![screen shot 2016-04-28 at 6 11 58 pm](https://cloud.githubusercontent.com/assets/8630999/14922964/535bcac4-0def-11e6-8695-2a80b0d3f8ac.png)
![screen shot 2016-04-29 at 8 49 56 am](https://cloud.githubusercontent.com/assets/8630999/14922967/56a8ab02-0def-11e6-8fc8-17492adbec74.png)

**Comment by catachresis** [comment]
Yeah, we haven't done a full migration of all the blog posts and it doesn't look like I can change the Disqus paths until that happens. Coming soon!

**Comment by seemala** [comment]
I figured something to that effect was the case :) Good luck and Godspeed
with full migration and updates!

-- 
C. M. Samala   /   c.samala@gmail.com
Google Voice   /  707 SAMALA C
Twitter  /  samala <http://twitter.com/samala>
